src/lists/list-item.jsx

```diff
@@ -407,6 +407,7 @@ const ListItem = React.createClass({
     const {
       innerDivStyle,
       style,
+      ...other,
```

Comment:
@oliviertassinari I might be confused, but after comparing the two ...other variables the one from line [364](https://github.com/callemall/material-ui/blob/master/src/lists/list-item.jsx#L364) (this being the one declared in the render function) when the enhanced button is created and the ...other variable on line  [410](https://github.com/callemall/material-ui/blob/master/src/lists/list-item.jsx#L410) I am seeing the same value. Any direction is appreciated, thanks.